### PR TITLE
Fix filtering in product imports

### PR DESCRIPTION
**Pull request by damien-carcel**
## Description

This PR fixes a EE bug (revealed by a broken integration test).

When importing a product without having the rights to even view this product, the import stop with an error. This is a wrong behavior introduced very recently (it is OK in 2.0.1), as the import should only skip the product with a warning.

The bug was caused by a filter that thrown an exception it didn't previously. This exception was only catch during product retrieval from database (after the data filtering).

The fix consists in catching the two possible exceptions at once in the same `try/catch`.

## Definition Of Done

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Todo
| Added Behats                      | -
| Added integration tests           | -
| Changelog updated                 | -
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: src/Pim/Component/Connector/Processor/Denormalization/ProductProcessor.php [hunk 1]
```diff
@@ -101,12 +102,17 @@ public function process($item)
...
-        $item = $this->productAttributeFilter->filter($item);
 
-        $familyCode = $this->getFamilyCode($item);
-        $filteredItem = $this->filterItemData($item);
+        try {
+            $item = $this->productAttributeFilter->filter($item);
 
-        $product = $this->findOrCreateProduct($identifier, $familyCode, $item, $parent);
+            $familyCode = $this->getFamilyCode($item);
+            $filteredItem = $this->filterItemData($item);
```
Comment (damien-carcel) [inline src/Pim/Component/Connector/Processor/Denormalization/ProductProcessor.php]: The exception can be thrown here

File: src/Pim/Component/Connector/Processor/Denormalization/ProductProcessor.php [hunk 2]
```diff
@@ -101,12 +102,17 @@ public function process($item)
...
+
+            $product = $this->findOrCreateProduct($identifier, $familyCode, $parent);
```
Comment (damien-carcel) [inline src/Pim/Component/Connector/Processor/Denormalization/ProductProcessor.php]: but also here.

File: src/Pim/Component/Connector/Processor/Denormalization/ProductProcessor.php [hunk 3]
```diff
@@ -13,6 +13,7 @@
 use Pim\Component\Catalog\Model\ProductInterface;
 use Pim\Component\Catalog\Model\VariantProductInterface;
 use Pim\Component\Catalog\ProductModel\Filter\AttributeFilterInterface;
+use PimEnterprise\Component\Security\Exception\ResourceAccessDeniedException;
```
Comment (skeleton) [inline src/Pim/Component/Connector/Processor/Denormalization/ProductProcessor.php]: nope, it's a EE exception. Use Symfony\Component\Security\Core\Exception\AccessDeniedException please

Comment (phaseinducer) [inline src/Pim/Component/Connector/Processor/Denormalization/ProductProcessor.php]: Nope, you don't know that as it depends on the implementation of your dependencies. And you're not allowed to add a `use` from CE to EE.

**Pull request merged**